skimage/morphology/binary.py

```diff
@@ -68,7 +96,10 @@ def binary_dilation(image, footprint=None, out=None):
         Binary input image.
     footprint : ndarray, optional
         The neighborhood expressed as a 2-D array of 1's and 0's.
-        If None, use a cross-shaped footprint (connectivity=1).
+        If None, use a cross-shaped footprint (connectivity=1). This can also
+        be a sequence of 2-tuples where the first element of each 2-tuple is a
+        footprint and the second element as an integer describing the number of
+        times it should be iterated.
```

Comment:
This:
> This can also be a sequence of 2-tuples where the first element of each 2-tuple is a footprint and the second element as an integer describing the number of times it should be iterated.

Could be in the `Notes`, like:
> `footprint` can also be a sequence (...)

And an example of that. What do you think?